grammars/asciidoc.cson

```diff
@@ -64,17 +80,123 @@
     'begin': '(^/{4,}$)'
     'end': '\\1'
     'name': 'comment.block.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches example block
+  #
+  # Examples
+  #
+  #   ====
+  #   A multi-line example.
+  #
+  #   Notice it's a delimited block.
+  #   ====
+  {
+    'begin': '^(={4,})+?$'
+    'end': '\\1'
+    'name': 'markup.block.example.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches sidebar block
+  #
+  # Examples
+  #
+  # ***********************************
+  # As an aside, I love Asciidoc
+  # ***********************************
+  {
+    'begin': '(^\\*{4,}$)'
+    'end': '\\1'
+    'name': 'markup.block.sidebar.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches literal block
+  #
+  # Examples
+  #
+  # ...................................
+  # Daleks EXTERMINATE in monospace!
+  # ...................................
+  {
+    'begin': '(^\\.{4,}$)'
+    'end': '\\1'
+    'name': 'markup.block.literal.asciidoc'
+    'patterns': [
+      { 'include': [ '#inline', '#codeBlocks'] }
```

Comment:
That's fair. I'll test both of the comments in this file against the book I'm writing (my test case) and report back either with a revised pull request or an illustration.